Source/Xamarin/Prism.Forms/Navigation/PageNavigationService.cs

```diff
@@ -300,7 +299,7 @@ Task<INavigationResult> IPlatformNavigationService.NavigateAsync(Uri uri, INavig
         /// <param name="animated">If <c>true</c> the transition is animated, if <c>false</c> there is no animation on transition.</param>
         /// <remarks>Navigation parameters can be provided in the Uri and by using the <paramref name="parameters"/>.</remarks>
         /// <example>
```

Comment:
No need to provide and example on this method as this is a protected method only meant for implementation by derived classes.